simplq/src/components/pages/Admin/index.jsx

```diff
@@ -66,7 +68,15 @@ export default (props) => {
 
   const HeaderSection = () => (
     <div className={styles['header-bar']}>
-      <Header className={styles['header']}>{queueName}</Header>
+      <div className={styles['header-title']}>
+        <Header className={styles['header']}>Rooftop Cafê Queue</Header>
```

Comment:
Let's keep both title and subtitle as `{queueName}`

Reply:
So the App name should not be displayed? I basically tried to implement the design from this sheet.

Reply:
We should be displaying the queue name in title. 

Anyone can create a queue. "Rooftop Cafê Queue" was just an example queue name in the design

Reply:
Ah okay, sorry, I misunderstood that. So is there a prop I could use as a description for the queue?

Reply:
It doesn't come from the backend yet. Lets either keep both the title and description as queueName, or we can add a placeholder text for now

Reply:
I added a description state variable with placeholder text.